src/lava/lib/dl/netx/hdf5.py

```diff
@@ -91,59 +95,60 @@ def get_neuron_params(
             The Lava process that implements the neuron described.
         """
         neuron_type = neuron_config['type']
+        num_message_bits = None
+        if 'messageBits' in neuron_config.keys():
+            num_message_bits = neuron_config['messageBits']
         if neuron_type in ['LOIHI', 'CUBA']:
+            if num_message_bits is None:
+                num_message_bits = 0  # default value
             neuron_process = LIF
-            neuron_params = {
-                'neuron_proc': neuron_process,
-                'vth': neuron_config['vThMant'],
-                'du': neuron_config['iDecay'] - 1,
-                'dv': neuron_config['vDecay'],
-                'bias_exp': 6,
-                'use_graded_spikes': False,
-            }
+            neuron_params = {'neuron_proc': neuron_process,
+                             # Key that actually represents bias in the
+                             # neuron_process
+                             'bias_key': 'bias_mant',
+                             # Rest of the neuron params
+                             'vth': neuron_config['vThMant'],
+                             'du': neuron_config['iDecay'] - 1,
+                             'dv': neuron_config['vDecay'],
+                             'bias_exp': 6,
+                             'num_message_bits': num_message_bits}
             return neuron_params
         elif neuron_type in ['SDNN']:
+            if num_message_bits is None:
+                num_message_bits = 16  # default value
             if input is True:
                 # If it is an input layer (input is true) use delta process.
```

Comment:
Could simplify comment to "Use delta process.", under don't repeat yourself, the code in 119 is redundant with half of the comment.